src/features/editable-table/index.tsx

```diff
@@ -0,0 +1,146 @@
+import { localStorageController } from '@/shared/utils';
+import { VariablesData } from '@/types/types';
+import { Button, Empty, Flex, Form, Popconfirm, Space, Table, TableProps, Typography } from 'antd';
+import { useEffect, useState } from 'react';
+import EditableCell from './editable-cell';
+import TableOperation from './operation';
+import { useTranslations } from 'next-intl';
+const { Text } = Typography;
+
```

Comment:
```suggestion
import '@ant-design/v5-patch-for-react-19';

```

Import that patch to get rid of the console error